### PR TITLE
Add featured for new 'ledgers'

### PR DESCRIPTION
**Pull request by humdingerb**
* Save window size/pos as attributes per ledger
* Install a CapitalBe ledger MIME type
* Add recent ledger submenu to "Open" menu
* Accept dropped ledgers

### CODE REVIEW

File: src/App.cpp [hunk 1]
```diff
@@ -78,6 +81,20 @@ App::AboutRequested()
 }
 
 
+void
+App::RefsReceived(BMessage* msg)
+{
+	entry_ref ref;
+	if (msg->FindRef("refs", &ref) == B_OK) {
```
Comment (korli) [inline src/App.cpp:88]: When fFile ist already set, fFile will be leaked.

File: src/App.h [hunk 2]
```diff
@@ -24,13 +25,15 @@ class App : public BApplication {
...
 
 private:
-	void ShowAlert(BString text);
-	void ShowMainWindow(BPath path);
+	void _InstallMimeType();
+	void _ShowAlert(BString text);
+	void _ShowMainWindow(BPath path);
 
 	MainWindow* fMainWindow;
+	BPath* fFile;
```
Comment (korli) [inline src/App.h:36]: Maybe switch the pointer to an instance.

File: src/App.cpp [hunk 3]
```diff
@@ -146,8 +145,8 @@ App::ReadyToRun()
 
 	LoadPreferences();
 
-	if (fFile != NULL) {
-		_ShowMainWindow(*fFile);
+	if (fFile.InitCheck() == B_OK) {
```
Comment (korli) [inline src/App.cpp:148]: I think, you can keep != NULL :)

Comment (korli) [reply src/App.cpp:148]: BPath has operators != und == , so maybe if (fFile != NULL)  could also work.

Comment (humdingerb) [reply src/App.cpp:148]: I tried, it does work. But since there are plenty InitCheck()s around already in the code, I'll just keep it here as well.

Comment (korli) [reply src/App.cpp:148]: OK!

**Pull request merged**